### ISSUE TITLE
Can I get the middle representation of model?

### ISSUE THREAD

**Issue by DawnJYe** [bytedance/DEADiff#8]
Hello！I want to decouple style information from image features. From the inference code you released, can I only obtain the decoupled output of QFormer, rather than the final image generated after adding text?

**Comment by Tianhao-Qi** [comment]
Yes, you can refer to the function [get_learned_conditioning](https://github.com/bytedance/DEADiff/blob/main/ldm/models/diffusion/blip_diffusion.py#L255-L314) in ldm/models/diffusion/blip_diffusion.py.

**Comment by DawnJYe** [comment]
OK，thanks for your answer.Does the return value of this function “encoder_hidden_states” represent decoupled style features?

**Comment by DawnJYe** [comment]
Oh，I think it may be the variable named "style_ctx_embeddings" which is attained by function "forward_ctx_embeddings",right?
